who.py

```diff
@@ -281,6 +281,28 @@ def _populate_indicator_data_db(self):
             self._session.commit()
             logger.info(f"Done indicator {indicator_name}")
 
+    @staticmethod
+    def get_showcase(
+        retriever, country_iso3, country_name, slugified_name, alltags
+    ):
+        try:
+            lower_iso3 = country_iso3.lower()
+            url = f"https://www.who.int/countries/{lower_iso3}/en/"
+            retriever.download_file(url)
+            showcase = Showcase(
+                {
+                    "name": f"{slugified_name}-showcase",
+                    "title": f"Indicators for {country_name}",
+                    "notes": f"Health indicators for {country_name}",
+                    "url": url,
+                    "image_url": f"https://www.who.int/sysmedia/images/countries/{lower_iso3}.gif",
```

Comment:
Oops I should have noted earlier, after testing I found that this URL has changed to:
`https://cdn.who.int/media/images/default-source/countries-overview/flags/{lower_iso3}.jpg`